datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/KafkaConnectorTask.java

```diff
@@ -70,12 +70,12 @@ static Properties getKafkaConsumerProperties(Properties consumerProps, String gr
     String bootstrapValue = csv.toString();
 
     Properties props = new Properties();
-    props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapValue);
```

Comment:
While it makes sense to make such change, could you confirm our existing consumer settings in the server config don't reply on such override?